google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -104,6 +108,61 @@ def _wrap_callback_errors(
         on_callback_error(exc)
 
 
+def _get_ack_errors(exc: exceptions.GoogleAPICallError) -> Optional["containers.ScalarMap"]:
+    if not exc.response:
+        _LOGGER.debug("No response obj in errored RPC call.")
+        return None
+    try:
+        status = rpc_status.from_call(exc.response)
+        if not status:
+            _LOGGER.debug("Unable to get status of errored RPC.")
+            return None
+        for detail in status.details:
+            if detail.Is(ErrorInfo.DESCRIPTOR):
+                info = ErrorInfo()
+                if not detail.Unpack(info):
+                    _LOGGER.debug("Unable to unpack ErrorInfo.")
+                    return None
+                return info.metadata
+    # Possible "If the gRPC call’s code or details are inconsistent
+    # with the status code and message inside of the
+    # google.rpc.status.Status"
+    except ValueError:
+        _LOGGER.debug( "ValueError when parsing ErrorInfo.", exc_info=True)
+        return None
+
+def _process_errors(future_reqs_dict: "containers.ScalarMap",
+                    errors_dict: Optional["containers.ScalarMap"]):
+    """Process errors returned by the server in `errors_dict` by completing
+    the request futures in `future_reqs_dict` (with a success or exception) or
+    returning requests for further retries.
+    """
```

Comment:
(nit) This docstring formatting does not follow the common form and is likely to render strangely in the docs.

Reply:
What's the conventional format? Are there some docs I can read about this?

Reply:
One way is to check the existing methods, but I also found [this guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods).